src/app/components/confirmDelete/ConfirmDelete.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@mui/material';
+
+interface ConfirmDeleteDialogProps {
+    open: boolean;
+    onClose: () => void;
+    onConfirm: () => void;
+  }
+
+const ConfirmDelete: React.FC<ConfirmDeleteDialogProps> = ({open, onClose, onConfirm }) => {
+    return (
+        <Dialog open={open} onClose={onClose}>
+          <DialogTitle>Delete Project</DialogTitle>
+          <DialogContent>
+            <DialogContentText>
+            Are you sure you want to Delete this Project?
```

Comment:
make the massege from a prop to make more abstract and be able to use it in multiple places